anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+    from gempython.utils.nesteddict import nesteddict as ndict
+    from gempython.gemplotting.mapping.chamberInfo import chamber_iEta2VFATPos
+    from gempython.gemplotting.mapping.chamberInfo import chamber_vfatPos2iEtaiPhi as vfat_to_etaphi
+    from gempython.gemplotting.utils.anautilities import make3x8Canvas, saveSummaryByiEta, getMapping
+    from gempython.gemplotting.utils.anaoptions import parser
+    import pkg_resources
+
+    parser.set_defaults(outfilename="sbitReadOut.root")
+    (options, args) = parser.parse_args()
+    path = options.filename
+    size = ((options.GEBtype).lower())
+
+    if not os.path.isdir(path):
+        raise OSError(2, "No such file or directory", path)
+    else:
+        pass
+
+    if size not in ('long', 'short'):
+        raise AssertionError("Invalid value of GEBtype")
+    else:
+        pass
+
+    filename = options.filename+"/sbitReadOut"
+    outfilename = options.outfilename
+    print("Analyzing: '%s'" % path)
+    os.system("mkdir "+filename)
+
+    """
+    At the moment the output of the sbitReadout.py are .dat files with headers that will automatically fill the ROOT TTree. 
+    However ther is one last ':' in the first line that shouldn't be there; consequently the ReadFile function is not able to understand the header. So before reading these .dat files, one has to be sure to remove the ':'
+    To achieve this the following command is needed (removes the last : in the header of all .dat files in the input path)
+    """
+    os.system("find "+path+" -iname \*.dat -exec sed -i 's/7\/i:/7\/i/g' {} \;")
+
+    # Set default histo behavior
+    r.TH1.SetDefaultSumw2(False)
+    r.gROOT.SetBatch(True)
+    r.gStyle.SetOptStat(1111111)
+
+    mappath = pkg_resources.resource_filename(
+        'gempython.gemplotting', 'mapping/')
+    # Loading the dictionary with the mapping
+    vfat_ch_strips = getMapping(
+        mappath+size+"ChannelMap_VFAT3-HV3b-V1_VFAT3-HV3b-V2.txt", False)
+    print("\nVFAT channels to strips "+size+":\t MAP loaded")
+
+    # Loading and reversing the dictionary with (eta , phi) <-> vfatN
+    etaphi_to_vfat = ndict()
+    for i in range(1, 9):
+        etaphi_to_vfat[i] = {v: k for k,
+                             v in chamber_iEta2VFATPos[i].iteritems()}
+
+    """
+        Now it's time to load all the input files and to merge them into one root TTree
+        The TTree file it's going to auto extend each time a new file is found
+        A TFile it's going to hold this TTree
+
+        """
+    # Creating the output File and TTree
+    outF = r.TFile(filename+'/'+outfilename, 'recreate')
+    inT = r.TTree('Packed', 'Tree Holding packed raw data')
+
+    # searching for all the files with this format and adding them to the TTree
+    start_time = time.time()
+    print ("\nReading .dat files from the folder '%s'" % path)
+    for idx, file in enumerate(glob.glob(path+'/sbitReadOut_run*.dat')):
+        inT.ReadFile(file)
+        # inT.Fill()
```

Comment:
Please address [this comment](
https://github.com/cms-gem-daq-project/vfatqc-python-scripts/pull/208#discussion_r226921101) by @jsturdy

Reply:
Fixed as suggested in the comment